tests/system/v1/test_reader_v1.py

```diff
@@ -27,6 +28,14 @@
 from google.cloud.bigquery_storage import types
 
 
+# Skip the test cases using bigquery client for mTLS testing since it
+# doesn't have mTLS support yet.
+skip_for_mtls = pytest.mark.skipif(
```

Comment:
I'm a bit concerned that if we add system tests it'll be a bit difficult to determine / remember to use this `skip_for_mtls` decorator.

Rather than annotating each method, how about we modify the function-scope fixtures that use the BigQuery client? Those fixtures could call [pytest.skip](https://docs.pytest.org/en/stable/reference.html?highlight=skip#pytest.skip) in the body if `use_mts` is true.

Reply:
done